opentelemetry-sdk/src/trace/span.rs

```diff
@@ -44,7 +44,7 @@ pub(crate) struct SpanData {
     /// Span events
     pub(crate) events: crate::trace::EvictedQueue<trace::Event>,
     /// Span Links
-    pub(crate) links: crate::trace::EvictedQueue<trace::Link>,
+    pub(crate) span_links: crate::trace::SpanLinks,
```

Comment:
fwiw, I liked `links` as the name without the `span_` prefix as everything in this struct relates to a span away.  e.g. `attributes` are not called `span_attributes` here.

Reply:
Yea I'd second that, what's the value from a user perspective?